px.go

```diff
@@ -7,10 +7,13 @@ import (
 	"net/http"
 	"os"
 	"sort"
+	"net/url"
+	"strconv"
+)
 
-	"strings"
-
-	"github.com/joho/godotenv"
+const (
+	pxApi = "https://api.500px.com/v1/photos/search/?"
+	rpp = 50
```

Comment:
Correct me if I'm wrong @qtoyoda I think it's number of photos we get from 500px per tag per request